src/couch_replicator/test/couch_replicator_compact_tests.erl

```diff
@@ -124,16 +128,25 @@ check_active_tasks(RepPid, {BaseId, Ext} = _RepId, Src, Tgt) ->
     Pending = couch_util:get_value(changes_pending, RepTask),
     ?assert(is_integer(Pending)).
 
-wait_for_replicator(Pid) ->
+
+get_pid(RepId) ->
```

Comment:
It looks like it is the same function as `couch_replicator_test_helper:get_pid/1`. Could we reuse the one from test_helper?